.npmignore

```diff
@@ -0,0 +1,2 @@
+.eslintrc.json
+__tests__
```

Comment:
We shouldn't need `.npmignore` because we have `files` section in package.json.

Reply:
Indeed